addons/mail/static/src/models/emoji_search_bar_view.js

```diff
@@ -79,6 +79,9 @@ registerModel({
         }),
         placeholder: attr({
             compute() {
+                if (this.emojiPickerView.hoveredEmojiPickerView) {
+                    return this.emojiPickerView.hoveredEmojiPickerView.emoji.shortcodes
```

Comment:
Don't forget to put a semicolon at end of line.

Semicolons are not mandatory in JavaScript code, but it is generally considered good practice to put them anyway. The reason is that rules for Automatic Semicolon Insertion (ASI) are sometimes a bit counterintuitive, which can lead to unexpected results. Here's an example:

```js
console.log("Odoo names:")

["TinyERP", "OpenERP", "Odoo"].forEach(name => console.log(name))
```
```
Uncaught TypeError: can't access property "Odoo", console.log(...) is undefined
```
In this example, JavaScript interprets the brackets as a property accessor rather than an array literal. In other words, it looks like this to the JavaScript parser:
```js
console.log("Odoo names:")["TinyERP", "OpenERP", "Odoo"].forEach(name => console.log(name))
```
Adding semicolons would remove the ambiguity and thus fix the code.